setup.py

```diff
@@ -35,10 +35,18 @@ def read(fname):
       author_email='matti.pastell@helsinki.fi',
       url='http://mpastell.com/pweave',
       packages=['pweave', 'pweave.themes', 'pweave.formatters', 'pweave.processors', 'pweave.bokeh'],
-      install_requires = ['markdown', 'pygments', 'ipython', 'nbformat',
-                          'nbconvert', 'jupyter_client', 'ipykernel'],
-      extras_require = {'test': ['scipy', 'matplotlib', 'coverage',
-                                   'ipython', 'nose', 'notebook'],
+      install_requires = ['markdown',
+                          'pygments',
+                          'ipython<=6.2.0',
```

Comment:
Why <=6.2.0? A comment is necessary.

Reply:
the interface to the kernel changes after 6.2 and pweave is not compatible with the newer interface

Reply:
I commented elsewhere that pinning a dep  with <= is tech debt that we need to eventually pay. In that case, it is to fix tests so it may be a trade-off we are willing to make, but just to have better logging, I am skeptical. If you had a link to this interface changes I'd like to take a look. How bad can it be between two consecutive minor versions? Shouldn't we just bit the bullet and restore compat?

Reply:
> the interface to the kernel changes after 6.2 and pweave is not compatible with the newer interface

That is definitely a major issue. Do we have it listed anywhere?

Reply:
There's nothing about this PR that causes pweave to be incompatible with newer versions of of ipython. That "tech debt" exists regardless of whether you pin a version. 

However, if you would like to address compatibility issue before considering this PR, I quite understand. Unfortunately, I don't remember the details of this as it has been many months since I looked at this code.

Reply:
I think it will be enough if you start an issue on that and refer it in a comment by that dependency.
Also mention that walkarround in the respective issue, so we can pay the debt when we address it.

Reply:
I see your point. You are just recognizing that a problem exists and working around it, and I have confused the work around with the problem. I stand corrected. So we should accept the workaround for now and open an issue to move to a more permanent solution (paying the tech debt).

Reply:
see #134